assignments/003-illumina-sequencing-questions/illumina-seq-homework.md

```diff
@@ -15,42 +15,47 @@ fork of the course repo on GitHub then send me a pull request.
 
 * Q1: True or False. This animation is showing that each of the two "fragments" is a double-stranded segment of DNA. (_from 30.0 to 33.0 seconds in the video_).
 
-
+FALSE.
 
 * Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
 
 
 
 * Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
 
-
+The sequencing binding sites are  oligonucleotides that attach to a template polymerase strand and provide a free 3’ hydroxyl group for the initiation of synthesis.
 
 * Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
 
-
+5' end - the polymerase travels from 5' to 3'. 
```

Comment:
The strand being synthesized is attached via its 5' end, but the template is its reverse complement, so it is attached by the 3' end.